arch/risc-v/src/qemu-rv/qemu_rv_start.c

```diff
@@ -154,10 +167,27 @@ void qemu_rv_start_s(int mhartid)
   extern void __trap_vec(void);
   WRITE_CSR(stvec, (uintptr_t)__trap_vec);
 
+  /* Set the trap vector for M-mode */
+
+  extern void __trap_vec_m(void);
+  WRITE_CSR(mtvec, (uintptr_t)__trap_vec_m);
+
+  /* Initialize mtimer before entering to S-mode */
+
+  extern void up_mtimer_initialize(void);
```

Comment:
Maybe all the externs can be grouped at the upper part of the file?

Reply:
Done